linux_lib/libcrystalhd/libcrystalhd_parser.cpp

```diff
@@ -657,8 +654,8 @@ BC_STATUS DtsAddH264SCode(HANDLE hDevice, uint8_t **ppBuffer, uint32_t *pUlDataS
 				//Succeeded
 
 				//Check NAL Unit Type
-				pNALU = pStart + Ctx->VidParams.StartCodeSz;
-				ulNalSize = Ctx->PESConvParams.m_lStartCodeDataSize;
+				// pNALU = pStart + Ctx->VidParams.StartCodeSz; // uint8_t *pNALU = NULL;
+			        // ulNalSize = Ctx->PESConvParams.m_lStartCodeDataSize; // uint32_t ulNalSize = 0;
```

Comment:
Indeed these variables are defined and useless.

The 'compat' layer was very useful in the past when the x86_64 was not the reference. Today I still have a working i686 distribution with some packages not in-sync with their x86_64 counterparts. This is still useful for testing, the i686 packages will run perfectly on the x86_64 kernel. We have too the x32 abi (x86_64 instructions with 32 bits pointer), this layer allows the compatibility too.